hack-resources/get-started.MD

```diff
@@ -3,15 +3,11 @@
 
 ## Set up local environment
 
-1. Clone the repository to your local machine and create a your new working branch with "alias/feature"
-2. Open the project in VS Code and reopen the project in dev container. VS Code will ask for it 
-3. Rename the file `.env.sample` to `.env` and update the values for the environment variables
-4. Run in VS Code Terminal 'pip install -r requirements.txt' this will install additional dependencies required for the project
-5. Az login in the terminal and select the subscription(ms-ignite-24)
-6. Add your account to cosmos DB built in role <br>
-    `az cosmosdb sql role assignment create --account-name cosmos-contoso-q5ezmhyg2mjqw --resource-group rg-contosochat --scope "/" --principal-id [USER_PRINCIPAL-ID] --role-definition-id 00000000-0000-0000-0000-000000000001 --subscription [SUBCRIPTION_ID]` 
-
-    `az cosmosdb sql role assignment create --account-name cosmos-contoso-q5ezmhyg2mjqw --resource-group rg-contosochat --scope "/" --principal-id [USER_PRINCIPAL-ID] --role-definition-id 00000000-0000-0000-0000-000000000002 --subscription [SUBCRIPTION_ID]` 
-
-7. Hit F5 to run the project in debug mode select `Python Debugger: FastAPI` as the configuration
-8. Open [api-test.http](api-test.http) and click on `Send Request` to test the API
+1. Clone the repository to your local machine and create a your new working branch with "alias/feature".
+1. Open the project in VS Code and reopen the project in dev container. VS Code will ask for it.
+1. Rename the file `.env.sample` to `.env` and update the values for the environment variables.
+1. Run in VS Code Terminal 'pip install -r requirements.txt' this will install additional dependencies required for the project.
+1. In the terminal, use the Azure CLI `az login` command to log into Azure.  Select the **ms-ignite-24** subscription.
+1. Run the `setup-cosmos-rbac.sh` script from the **hack-resources** folder to set up the needed permissions for Cosmos DB.
+1. Hit F5 to run the project in debug mode select `Python Debugger: FastAPI` as the configuration.
+1. Open [api-test.http](api-test.http) and click on `Send Request` to test the API.

```

Comment:
This assumes the Simple REST Client extension is installed.  Should we note that?